demo/site/src/util/graphQLClient.ts

```diff
@@ -0,0 +1,10 @@
+import { createFetchWithDefaults, createFetchWithPreviewHeaders, createGraphQLFetch as createGraphQLFetchLibrary } from "@comet/cms-site";
+import { SitePreviewData } from "@src/app/api/site-preview/route";
+
+export const graphQLApiUrl = `${typeof window === "undefined" ? process.env.API_URL_INTERNAL : process.env.NEXT_PUBLIC_API_URL}/graphql`;
```

Comment:
Why do you export this?

Reply:
mainly because I also did it here https://github.com/vivid-planet/comet/pull/1314/files#diff-40fa500a23f6fba443b21f47fa38796c2b8c8d0cb0c97b140d09a3b3a7d40e23R8

But so far it's useless